### PR TITLE
WobbleServer as resource

### PR DESCRIPTION
**Pull request by Velrok**
can be used in with statements

Based on #11 . Merge that first.

### CODE REVIEW

**Comment by Velrok** [comment]
Based on #11 . Merge that first.

File: wobble/__init__.py [hunk 1]
```diff
@@ -34,6 +43,18 @@ def __init__(self, api_endpoint='http://wobble.moinz.de/api/endpoint.php',
...
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exception=None, error=None, traceback=None):
+        if exception:
+            logging.error(exception)
+            logging.error(error)
+            logging.error(traceback)
+
+        self.user_signout()
```
Comment (zeisss) [inline wobble/__init__.py:55]: So, what if the login failed?

**Comment by Uepsilon** [comment]
I do not think it is a good idea to base pull-request on previous, still pending pull request - except it is needed but then it should be one request instead of many. for example, i think @ZeissS is right and we have to rethink the signout in the exit-method. so basically i'd like to postpone this pull-request and do not merge it into the master bevor fixing that. that leaves me with the decision whether to merge the others or not. for now the pull request are kinda small and easy to check but that might not be the case later

**Comment by Velrok** [comment]
Very valid points!

However given the very small size and fun / freetime nature of this project I would say let us merge this and learn for the future -> no dependencies between pull requests.

Am Sonntag, 17. März 2013 | KW 11 um 10:15 schrieb Uepsilon:

**Comment by Uepsilon** [comment]
I'll at least create an Issue so we will not forget about that ;)

**Comment by Velrok** [comment]
+1  

Am Sonntag, 17. März 2013 | KW 11 um 10:38 schrieb Uepsilon:

**Pull request merged**